From: Ximin Luo <infinity0@debian.org>
Date: Wed, 7 Dec 2016 18:09:01 +0100
Subject: Call lessc with --source-map-basepath option for reproducibility

---
 setupbase.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- jupyter-notebook.orig/setupbase.py
+++ jupyter-notebook/setupbase.py
@@ -484,7 +484,7 @@
         for src, dst in zip(self.sources, self.targets):
             try:
                 run(['lessc',
-                    '--source-map',
+                    '--source-map', '--source-map-basepath='+os.path.dirname(__file__),
                     '--include-path=%s' % pipes.quote(static),
                     src,
                     dst,
