From: Gordon Ball <gordon@chronitis.net>
Date: Sun, 25 Jun 2017 11:53:23 +0200
Subject: Ignore errors in documentation notebooks during build

---
 docs/source/conf.py | 1 +
 1 file changed, 1 insertion(+)

--- jupyter-notebook.orig/docs/source/conf.py
+++ jupyter-notebook/docs/source/conf.py
@@ -360,3 +360,4 @@
 
 # import before any doc is built, so _ is guaranteed to be injected
 import notebook.transutils
+nbsphinx_allow_errors = True
